cmd/utils/flags.go

```diff
@@ -1600,6 +1606,19 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 			cfg.EthDiscoveryURLs = SplitAndTrim(urls)
 		}
 	}
+	if ctx.GlobalIsSet(HTTPRpcTimeoutFlag.Name) {
+		arg := ctx.GlobalString(HTTPRpcTimeoutFlag.Name)
+		_, err := strconv.Atoi(arg)
+		if err == nil {
+			arg = arg + "ns"
+		}
+		timeout, err := time.ParseDuration(arg)
+		if err != nil {
+			log.Warn("Bad http.timeout setting", arg)
```

Comment:
Our log facilities are a bit peculiar: essentially it's on the form `log.Level( <message>, <key-name>, <value>, <key-name>, <value>)`. So it's always `1 + 2N` arguments. In this case, it would be appropriate with e.g. `log.Warn("Failed to parse http timeout duration", "err", err)`